### DISCUSSION TITLE
v4.7.2

### DISCUSSION THREAD

**Discussion by panva** [panva/openid-client#361] [Releases]
### Bug Fixes

* **typescript:** add types for 4.6.0 additions ([9064136](https://github.com/panva/node-openid-client/commit/9064136d959b5825f69b32344bbe165f12a10949))

<hr /><em>This discussion was created from the release <a href='https://github.com/panva/node-openid-client/releases/tag/v4.7.2'>v4.7.2</a>.</em>

**Comment by Frog-kt** [comment]
@panva 
I've been waiting for this! Thank you!!
